index.html

```diff
@@ -42,5 +41,182 @@ <h5>LET'S CONNECT</h5>
                 </div>
             </div>
         </section>
+        <section class="works">
+            <div class="works-flex">
+                <div class="works-flex-item work-image">
+                    <img src="assets/images/portfolio_snapshoot/snapshoot1.png" class="img-style" alt="tonic works" />
+                </div>
+                <div class="works-flex-item">
+                    <div class="intro-text-works">Tonic</div>
```

Comment:
- [ ] Hi, well done on implementing the design of this section really well. As per the requirements Each of the project titles should use a heading tag (`h2`, `h3`, `h4` e.t.c). This is very important because we want to have clean sematic HTML. in this case it make a lot more sense for a project title to have a heading tag. Thank you for understanding 🙏
something like this would be better:
```suggestion
                    <h3 class="intro-text-works">Tonic</h3>
```

- [ ] do this for all the other card titles as well 👍.